src/locales/en.json

```diff
@@ -1,171 +1,218 @@
-    },
-
-    "step-navigation" : {
-        "previous": "Change earlier step" ,
-        "next": ""
-    },
-    "free-culture-work": {
-        "yes": "Free Culture License",
-        "no": "Not a Free Culture License"
+        "richtext": {
+            "full-text": "{workTitle} {by} {creator}{licensed-text}{licenseName}",
+            "workTitle": "This work",
+            "by": "by",
+            "licensed-text": " is licensed under ",
+            "print-instructions": ". To view a copy of this license, visit {linkToLicenseDeed}."
```

Comment:
The period after `{linkToLicenseDeed}` could cause confusion in the plain text for print work or media attribution. This period, when included in the URL to the license deed causes a 404.

For example:
`https://creativecommons.org/publicdomain/zero/1.0` does not return 404, but
`https://creativecommons.org/publicdomain/zero/1.0.` does.

Reply:
Period removed.